SofaKernel/modules/SofaCore/src/sofa/core/behavior/BaseForceField.h

```diff
@@ -119,8 +112,9 @@ class SOFA_CORE_API BaseForceField : public virtual objectmodel::BaseObject
     /// by the dx vector with the given coefficients.
     ///
     /// This method computes
-    ///
-    ///            $ df += mFactor M dx + bFactor B dx + kFactor K dx $
+    /// \f[
+    ///            df += mFactor M dx + bFactor B dx + kFactor K dx
```

Comment:
No more `$` here, is it normal?

Reply:
Actually, I forgot to remove the `$` in the other equations. Thanks for pointing it out.